flink-connector/src/main/java/org/apache/flink/connector/delta/sink/writer/DeltaWriter.java

```diff
@@ -45,32 +59,310 @@
         implements SinkWriter<IN, DeltaCommittable, DeltaWriterBucketState>,
         Sink.ProcessingTimeService.ProcessingTimeCallback {
 
-    public DeltaWriter() {
+    private static final Logger LOG = LoggerFactory.getLogger(DeltaWriter.class);
+
+    ///////////////////////////////////////////////////////////////////////////
+    // FileSink-specific fields
+    ///////////////////////////////////////////////////////////////////////////
+
+    ///////////////////////////////////////////////////////////////////////////
+    // configuration fields
```

Comment:
if `configuration fields` is a "sub-header" of `FileSink-specific fields`, can you make the `\\\\\\\\...` shorter?
i.e.
```
///////////////////////////////////////////////////////////////////////////
// FileSink-specific fields
///////////////////////////////////////////////////////////////////////////
...
///////////////////////////////////////////////////
// configuration fields
///////////////////////////////////////////////////
```